docs/src/operate/registration/register.md

```diff
@@ -431,6 +433,74 @@ curl 'http://localhost:8000/tedge/entity-store/v1/entities?parent=device/child2/
 ]
 ```
 
+## Update entity twin data
+
+Update an exiting entity, adding new twin data fragments or removing existing fragments.
+
+**Endpoint**
+
+```
+PATCH /v1/entities/{topic-id}
+```
+
+**Payload**
+
+Update existing fragment: `name`, add new fragment: `hardware` and remove existing fragment: `maintenanceMode` with a `null` value:
+
+```json
+{
+    "name": "Child 01",
+    "new-fragment": {
+        "new-key": "new-value"
+    },
+    "extra-fragment": null
+}
+```
+
+**Example**
+
+```shell
+curl http://localhost:8000/tedge/entity-store/v1/entities/device/child01 \
+  -X PATCH \
+  -H "Content-Type: application/json" \
+  -d '{
+    "type": "Raspberry Pi 4",
+    "hardware": {
+        "serialNo": "98761234"
+    },
+    "maintenanceMode": null
+  }'
+```
```

Comment:
The example doesn't match the text.

```suggestion
Update existing fragment: `name`, add new fragment: `hardware` and remove existing fragment: `maintenanceMode` with a `null` value:


```shell
curl http://localhost:8000/tedge/entity-store/v1/entities/device/child01 \
  -X PATCH \
  -H "Content-Type: application/json" \
  -d '{
    "name": "Child 01",
    "hardware": {
        "serialNo": "98761234"
    },
    "maintenanceMode": null
  }'
```
```

Reply:
Resolved by 6e7c4c6